app/scripts/modules/core/cluster/filter/clusterFilter.service.spec.js

```diff
@@ -769,8 +769,8 @@ describe('Service: clusterFilterService', function () {
       application.clusters = clusterService.createServerGroupClusters(application.serverGroups.data);
       service.updateClusterGroups(application);
       $timeout.flush();
-      expect(ClusterFilterModel.groups[0].subgroups[0].subgroups[0].serverGroups[0]).toBe(this.serverGroup000);
-      expect(ClusterFilterModel.groups[0].subgroups[0].subgroups[0].serverGroups[1]).toBe(this.serverGroup001);
+      expect(ClusterFilterModel.groups[0].subgroups[0].subgroups[0].serverGroups[0]).toEqual(this.serverGroup000);
+      expect(ClusterFilterModel.groups[0].subgroups[0].subgroups[0].serverGroups[1]).toEqual(this.serverGroup001);
```

Comment:
It doesn't seem like these changes should be necessary.

IIRC, the purpose of this code - and these checks - is to verify that the object reference is unchanged. We rely on this behavior to let the UI know whether or not to re-render the server group on the page. Same thing below - we want to see that the object references are unchanged.

From looking at the changes in this PR, it seems like `toBe` should still work, but...maybe not?